mappings/net/minecraft/block/entity/SkullBlockEntity.mapping

```diff
@@ -9,8 +9,11 @@ CLASS net/minecraft/class_2631 net/minecraft/block/entity/SkullBlockEntity
 		ARG 1 pos
 		ARG 2 state
 	METHOD method_11333 setOwner (Lcom/mojang/authlib/GameProfile;)V
+		ARG 1 owner
 	METHOD method_11334 getOwner ()Lcom/mojang/authlib/GameProfile;
 	METHOD method_11335 loadProperties (Lcom/mojang/authlib/GameProfile;Ljava/util/function/Consumer;)V
+		ARG 0 owner
+		ARG 1 ownerConsumer
```

Comment:
This definitely has a better name. I guess it's like a `propertyUpdater` once the owner game profile has been retrieved from the profile repository?

Reply:
```suggestion
		ARG 1 callback
```